### DISCUSSION TITLE
Slack Integration with AutoGen

### DISCUSSION THREAD

**Discussion by gs-mteki** [microsoft/autogen#4001] [General]
Hey,

I am exploring/new to the AutoGen. From my understanding it's not that simple to integrated AutoGen with Slack or any messaging app. Specially with Human Inputs. Where Slack Bot ask the user to get specific information and executes the code once the all the information is in place like application name and environment.

For example, executing the kubectl command to check pods are running or not. After getting the application name and environment.

Am I right, or is there anything I am missing ?

**Comment by ekzhu** [comment]
There is a related discussion on this topic: https://github.com/microsoft/autogen/discussions/3915. In short, it is much easier with the v0.4 API. 

Hopefully we will have some samples to show an example integration with chat apps.
